From 1eccb76db85bdcad91cdca30cdde2bd20c8cf998 Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@cloud.ionos.com>
Date: Tue, 28 May 2019 14:03:38 +0200
Subject: [PATCH] btrfs_stats: Upgrade to Python 3

Python 2.7 will not be maintained past 2020. Therefore upgrade
`text_collector_examples/btrfs_stats.py` to Python 3.

Forwarded: https://github.com/prometheus/node_exporter/pull/1359
Signed-off-by: Benjamin Drung <benjamin.drung@cloud.ionos.com>
---
 text_collector_examples/btrfs_stats.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/text_collector_examples/btrfs_stats.py b/text_collector_examples/btrfs_stats.py
index b26bfd2..68e89a8 100755
--- a/text_collector_examples/btrfs_stats.py
+++ b/text_collector_examples/btrfs_stats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Collect per-device btrfs filesystem errors.
 # Designed to work on Debian and Centos 6 (with python2.6).
-- 
2.20.1

